addons/purchase/wizard/purchase_order_cancel.xml

```diff
@@ -0,0 +1,19 @@
+<odoo>
+    <record id="view_purchase_cancel_wizard" model="ir.ui.view">
+        <field name="name">purchase.cancel.wizard.form</field>
+        <field name="model">purchase.cancel.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Cancel Purchase Orders">
+                <group>
+                    <div class="o_form_label" colspan="2">
+                        Are you sure you want to cancel the <span><field name="purchase_count" class="oe_inline"/></span> selected RFQs ?
+                    </div>
+                </group>
+                <footer>
+                    <button string="Cancel RFQs" type="object" name="action_confirm" class="btn-primary"/>
+                    <button string="Discard" type="object" name="action_discard" class="btn-secondary"/>
+                </footer>
+            </form>
+        </field>
+    </record>
+</odoo>
```

Comment:
delete

Reply:
If you want a warning message. Add the button in the header of list view with a confirm argument